setup.py

```diff
@@ -66,7 +65,6 @@
 ext_kwargs = {'include_dirs': ['src']}  # We add np.get_include() later
 
 for modulename, other_sources, language in (
-        ('dipy.core.interpolation', [], 'c'),
```

Comment:
Same comment as above. You need to keep this

Reply:
Probably I got it from the old version. 
I will update now